app/browser/api/ledger.js

```diff
@@ -1836,6 +1841,7 @@ const onWalletProperties = (state, body) => {
   const balance = parseFloat(body.get('balance'))
   if (balance >= 0) {
     state = ledgerState.setInfoProp(state, 'balance', balance)
+    state = ledgerState.setInfoProp(state, 'userHasFunded', true)
```

Comment:
@jasonrsadler this only need to happen once right?

Reply:
correct. Once true, it's true forever